sdk/rust/oak/src/grpc/server.rs

```diff
@@ -16,27 +16,68 @@
 
 //! Functionality to help Oak Nodes create gRPC server pseudo-Nodes.
 
-use crate::{OakStatus, ReadHandle};
+use crate::{grpc::Invocation, io::Receiver, OakStatus};
+
+struct GrpcServerInit {
```

Comment:
Why is this message needed?

Reply:
To avoid passing handles around manually. Currently the interaction is very low level, we pass handles around using channel_write, but that's not really how channels are meant to be used. That's why I added a TODO on the gRPC server pseudo-node impl, to migrate that to using a proper encapsulated type, rather than reading individual handles and having assertions around them that they are of the correct number and shape. Basically currently we are doing the equivalent of reading bytes and putting them together, instead of using a proper serialization format like protobuf.

TL;DR: type safety == good; manually manipulating bytes + handles == bad